node_modules/oae-meetups/lib/restmodel.js

```diff
@@ -0,0 +1,14 @@
+/*!
```

Comment:
Let's keep it. I've taken a closer look now and it's okay to be empty since the `rest.js` file has no new types in `@Return`